src/sentry/migrations/0439_add_control_option.py

```diff
@@ -0,0 +1,46 @@
+# Generated by Django 2.2.28 on 2023-05-09 22:55
+
+import django.utils.timezone
+from django.db import migrations, models
+
+import sentry.db.models.fields.bounded
+import sentry.db.models.fields.picklefield
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production. For
+    # the most part, this should only be used for operations where it's safe to run the migration
+    # after your code has deployed. So this should not be used for most operations that alter the
+    # schema of a table.
+    # Here are some things that make sense to mark as dangerous:
+    # - Large data migrations. Typically we want these to be run manually by ops so that they can
+    #   be monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   have ops run this and not block the deploy. Note that while adding an index is a schema
+    #   change, it's completely safe to run the operation after the code has deployed.
+    is_dangerous = False
+
+    dependencies = [
+        ("sentry", "0438_break_inviter_fk_organizationmember"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
FYI, @owensk  is making changes to the schema of this table in saas 
https://github.com/getsentry/sentry/pull/47510/files#diff-30d47f3e1b7a360b6f72cb9f43b18f4dff7d12e3634d6570de941c2c44b82d72R28

That is in order to add a field to identify the channel we used to make the last update for each options as there are multiple conflicting channels today and the autoamted tool he is building needs to be able to identify drift. 

I think you two should sync depending which PR lands first.
- If this lands first, @owensk  you should update yours to make the changes to this table as well.
- IF @owensk  PR lands first this should be  modified to add the additional field.